client/components/UI/Button.tsx

```diff
@@ -1,41 +1,60 @@
 import React from 'react';
-import PropTypes from 'prop-types';
 import classNames from 'classnames';
 
 import {KEYCODES} from './constants';
 import {onEventCapture} from './utils';
 
 
-/**
- * @ngdoc react
- * @name Button
- * @description Generic Button component
- */
+interface IButtonProps {
```

Comment:
Just as a note, we should replace these with the ones from the UI Framework

Reply:
I can do that for this ticket if you mean replacing the button